Update the frame rate limiter description (#2872)
ppy/osu-wiki: wiki/Options/en.md

| Name | Description |
| --- | --- |
| `VSync` | Limits the game client to the refresh rate of your monitor. See explanation below for details. |

> | `Power Saving` | Limits the game client to 2x refresh rate. |

Reviewer: ```suggestion
| `Power Saving` | Limits the game client to twice the refresh rate of your monitor. |
```

> | `Optimal` | Limits the game client between 480~960fps based on 8x your monitor refresh rate. |

Reviewer: ```suggestion
| `Optimal` | Limits the game client to eight times the refresh rate of your monitor, usually between 480–960fps. |
```

> | `Optimal` | Limits the game client to eight times the refresh rate of your monitor, usually between 480–960fps. |

Reviewer 2: ```suggestion
| `Optimal` | Limits the game client to eight times the refresh rate of your monitor, capping at 960fps. |
```

| `Unlimited (gameplay)` | Limits the game client to the refresh rate of your monitor. See explanation below for details. |

- `VSync` explanation: In layman terms, using VSync will force the game wait for the entire frame to load before displaying it.
  - You may want to use this if you see some "tearing" (when the bottom portion of the game is lagging behind the upper portion of the game).
  - Despite what the point above stated, this may cause some lag or slowdowns because the game client has to wait for each frame to load.
- `Unlimited (gameplay)` explanation: Unlimited only applies to when you are playing a beatmap.

>   - When you are doing anything but playing a beatmap, `240fps` is used instead.

Reviewer: ```suggestion
  - When you are doing anything but playing a beatmap, the frame rate is limited to 240fps.
```

>   - When you are doing anything but playing a beatmap, the frame rate is limited to 240fps.

Reviewer 2: 2x monitor refresh rate now

Author: It seems to be still limiting to the 240fps, is it a 4x refresh rate? (cuttingedge 20191207.1)

Reviewer 2: 2x but lowest possible is 240, i guess

Reviewer 2: ```suggestion
  - When you are not playing a beatmap, the frame rate is limited to twice the refresh rate of your monitor, or 240fps, whichever is higher.
```

>   - This option is **not recommended**. Using unlimited frame limiter can lead to stutters.

Reviewer: ```suggestion
  - This option is **not recommended**. Using the `Unlimited (gameplay)` option can lead to stutters.
```

>   - By using the `Optimal` setting, you will be automatically set to a frame limiter which (if your hardware can reach) offers imperceivable changes in system (input to output) latency when compared to `unlimited`.

Reviewer: ```suggestion
  - Using the `Optimal` option instead offers imperceivable changes in system (input to output) latency when compared to `Unlimited (gameplay)`.
```

I think we can shorten this up to what I suggested. The `Optimal` setting was explained in the table above.
